crates/forge/tests/e2e/diagnostics_and_plugins.rs

```diff
@@ -1,127 +1,130 @@
-use super::common::runner::{setup_package, test_runner};
+use super::common::runner::{
+    setup_package_with_rpc_url_placeholders, test_runner, BASE_FILE_PATTERNS,
+};
 use indoc::formatdoc;
+use shared::test_utils::node_url::node_url_with_version;
 use shared::test_utils::output_assert::assert_stderr_contains;
 
 #[test]
 fn print_error_if_attributes_incorrect() {
-    let mock_tests_dir = setup_package("diagnostics_and_plugins");
-    let mock_tests_dir_path = mock_tests_dir.path().canonicalize().unwrap();
-    let mock_tests_dir_path_str = mock_tests_dir_path.to_str().unwrap();
-
+    let node_url = node_url_with_version().unwrap();
+    let mock_tests_dir =
+        setup_package_with_rpc_url_placeholders("diagnostics_and_plugins", BASE_FILE_PATTERNS);
```

Comment:
i think it should be done automatically, on every `setup_package`

Reply:
`setup_package` is called almost 50 times across all tests, wouldn't this be a bit of an unnecessary overhead?

Reply:
that's why i wanted to have a function which would only look for the "marked" filenames in order to replace anything. but it's a nitpick more than anything, so if you think it's not worth it, you can resolve